sass/routes/_settings.scss

```
@@ -365,23 +360,22 @@
                         }
 
                         .header__icon {
-                            @include opacity(0.5);
-
                             float: right;
+                            opacity: 0.5;
                         }
                     }
 
                     .theme-elements__body {
-                        @include border-radius(0 0 3px 3px);
-                        @include legacy-pie-clearfix;
-                        @include alpha-property(background-color, $white, 0.05);
-
                         max-height: 0;
                         padding: 0 0 0 24px;
                         margin: 0 20px 0 0;
+                        border-radius: 0 0 3px 3px;
                         overflow-y: hidden;
                         transition: all 0.4s ease-out;
 
+                        @include pie-clearfix;
+                        @include alpha-property(background-color, $white, 0.05);
```

Non blocking: white is white 😛. Worth just replacing with this?

```suggestion
                        background-color: rgba(255, 255, 255, 0.05);
```